src/lib.rs

```diff
@@ -1,6 +1,27 @@
+use elements::{BlockHash, Txid};
+
+pub mod avg_vbytes;
```

Comment:
Do we need to expose this publicly?

Reply:
Annoyingly still used in `waves` but it would be nice if we could get rid of it.

https://github.com/comit-network/waves/blob/extract-wallet/extension/wallet/src/protocol/make_create_swap_payload.rs#L126-L137